sycl/include/sycl/handler.hpp

```diff
@@ -1727,36 +1729,18 @@ class __SYCL_EXPORT handler {
   handler &operator=(const handler &) = delete;
   handler &operator=(handler &&) = delete;
 
+  // This is somewhat radical, but to make handler.hpp independtent from
+  // kernel_bundle.hpp, we define those methods within kernel_bundle.hpp
+  // header. Independence is needed in context of potential upcoming split of
+  // sycl.hpp so that users could do fine-grained include's, saving on
+  // compilation time by avoiding using headers for features they don't use.
```

Comment:
I've done the same in a few places already for previous splits. I think mine are in `queue.hpp`